### ISSUE TITLE
Invalid position given by 'no-multiple-empty-lines'

### ISSUE THREAD

**Issue by wawhal** [AtomLinter/linter-eslint#1140]
ESLint returned a point that did not exist in the document being edited.
Rule: `no-multiple-empty-lines`
Requested range: 84:1

Debug information:
```json
{
  "atomVersion": "1.22.1",
  "linterEslintVersion": "8.4.1",
  "linterEslintConfig": {
    "lintHtmlFiles": false,
    "useGlobalEslint": false,
    "showRuleIdInMessage": true,
    "disableWhenNoEslintConfig": true,
    "eslintrcPath": "",
    "globalNodePath": "",
    "advancedLocalNodeModules": "",
    "eslintRulesDirs": [],
    "disableEslintIgnore": false,
    "disableFSCache": false,
    "fixOnSave": false,
    "scopes": [
      "source.js",
      "source.jsx",
      "source.js.jsx",
      "source.babel",
      "source.js-semantic"
    ],
    "rulesToSilenceWhileTyping": [],
    "rulesToDisableWhileFixing": [],
    "ignoreFixableRulesWhileTyping": false
  },
  "eslintVersion": "1.10.3",
  "hoursSinceRestart": 1.7,
  "platform": "linux",
  "eslintType": "local project",
  "eslintPath": "/home/wawhal/workspace/platform/components/console/app/node_modules/eslint",
  "editorScopes": [
    "source.js.jsx",
    "meta.method-call.with-arguments.js",
    "meta.function.arrow.js",
    "punctuation.terminator.statement.js"
  ]
}
```

**Comment by IanVS** [comment]
Hi, this is due to a bug in the very old version of eslint that you are using.  Please update to version 4 (or version 5 which is in alpha) to fix it.

**Comment by Arcanemagus** [comment]
For reference: ([v1 -> v2](https://eslint.org/docs/user-guide/migrating-to-2.0.0), [v2 -> v3](http://eslint.org/docs/user-guide/migrating-to-3.0.0), [v3 -> v4](http://eslint.org/docs/user-guide/migrating-to-4.0.0)).

This particular rule was fixed in ESLint version [v3.5.0](http://eslint.org/blog/2016/09/eslint-v3.5.0-released).

Thanks for taking the time to report this though!

**Comment by wawhal** [comment]
There were 5 empty lines together in my code. The error went away when I removed 4 of them.

Thanks for the quick notice. I updated the version. Didn't even know it was that old haha.
